src/lib/three.js

```diff
@@ -19,10 +19,10 @@ if (THREE.Cache) {
 }
 
 // TODO: Eventually include these only if they are needed by a component.
-require('three/examples/js/loaders/DRACOLoader');  // THREE.DRACOLoader
-require('three/examples/js/loaders/GLTFLoader');  // THREE.GLTFLoader
-require('three/examples/js/loaders/OBJLoader');  // THREE.OBJLoader
-require('three/examples/js/loaders/MTLLoader');  // THREE.MTLLoader
+require('../../vendor/loaders/DRACOLoader');  // THREE.DRACOLoader
```

Comment:
The examples/js folder is still accessible / included, you don't have to pull in locally.

Reply:
Looks good beyond this.

Reply:
I changed because examples were no longer pulled after `npm install`

Reply:
<img width="433" alt="screen shot 2019-01-31 at 3 20 20 pm" src="https://user-images.githubusercontent.com/674727/52092232-bdf21c80-256b-11e9-862e-0221d4f268df.png">

Reply:
It does not work for me:
![image](https://d3vv6lp55qjaqc.cloudfront.net/items/3P1Z2v1A0C1x3r2U1s0h/Image%202019-02-01%20at%205.38.56%20PM.png?X-CloudApp-Visitor-Id=222920)